src/QUICConnection.ts

```diff
@@ -413,6 +396,37 @@ class QUICConnection {
     this.resolveClosedP = resolveClosedP;
   }
 
+  /**
+   * This is the source connection ID.
+   */
+  public get connectionId() {
+    return new QUICConnectionId(this.conn.sourceId());
+  }
+
+  /**
+   * This is the destination connection ID.
+   * This is only fully known after establishing the connection
+   */
+  @ready(new errors.ErrorQUICConnectionNotRunning())
+  public get connectionIdPeer() {
+    return new QUICConnectionId(this.conn.destinationId());
```

Comment:
You should make this zero copy. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array/Uint8Array

Reply:
This has been updated and now is doing a zero copy construction of `QUICConnectionId`.